Cargo.lock

```diff
@@ -2574,7 +2600,15 @@ name = "toml"
 version = "0.4.8"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 dependencies = [
- "serde 1.0.84 (registry+https://github.com/rust-lang/crates.io-index)",
+ "serde 1.0.104 (registry+https://github.com/rust-lang/crates.io-index)",
+]
+
+[[package]]
+name = "toml"
+version = "0.5.5"
```

Comment:
While you're upgrading, can you bump `toml` to 0.5 so it's not duplicated?

Reply:
Done.